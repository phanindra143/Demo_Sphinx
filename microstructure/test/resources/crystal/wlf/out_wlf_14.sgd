*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   114.0233
  !k_ij(1,2)=     0.5583
  !k_ij(2,2)=   114.4590
  !k_ij(1,3)=     0.7066
  !k_ij(2,3)=     0.5248
  !k_ij(3,3)=   113.8795
MP,Kxx,1,  114.02331
MP,Kyy,1,  114.45898
MP,Kzz,1,  113.87955
MP,Kxx,2,  114.45898
MP,Kyy,2,  114.02331
MP,Kzz,2,  113.87955
MP,Kxx,3,  114.45898
MP,Kyy,3,  114.02331
MP,Kzz,3,  113.87955
MP,Kxx,4,  114.02331
MP,Kyy,4,  114.45898
MP,Kzz,4,  113.87955
MP,Kxx,5,  114.02331
MP,Kyy,5,  114.45898
MP,Kzz,5,  113.87955
MP,Kxx,6,  114.45898
MP,Kyy,6,  114.02331
MP,Kzz,6,  113.87955
MP,Kxx,7,  114.45898
MP,Kyy,7,  114.02331
MP,Kzz,7,  113.87955
MP,Kxx,8,  114.02331
MP,Kyy,8,  114.45898
MP,Kzz,8,  113.87955
MP,Kxx,9,  113.87955
MP,Kyy,9,  114.02331
MP,Kzz,9,  114.45898
MP,Kxx,10,  114.02331
MP,Kyy,10,  113.87955
MP,Kzz,10,  114.45898
MP,Kxx,11,  114.02331
MP,Kyy,11,  113.87955
MP,Kzz,11,  114.45898
MP,Kxx,12,  113.87955
MP,Kyy,12,  114.02331
MP,Kzz,12,  114.45898
MP,Kxx,13,  113.87955
MP,Kyy,13,  114.02331
MP,Kzz,13,  114.45898
MP,Kxx,14,  114.02331
MP,Kyy,14,  113.87955
MP,Kzz,14,  114.45898
MP,Kxx,15,  114.02331
MP,Kyy,15,  113.87955
MP,Kzz,15,  114.45898
MP,Kxx,16,  113.87955
MP,Kyy,16,  114.02331
MP,Kzz,16,  114.45898
MP,Kxx,17,  114.45898
MP,Kyy,17,  113.87955
MP,Kzz,17,  114.02331
MP,Kxx,18,  113.87955
MP,Kyy,18,  114.45898
MP,Kzz,18,  114.02331
MP,Kxx,19,  113.87955
MP,Kyy,19,  114.45898
MP,Kzz,19,  114.02331
MP,Kxx,20,  114.45898
MP,Kyy,20,  113.87955
MP,Kzz,20,  114.02331
MP,Kxx,21,  114.45898
MP,Kyy,21,  113.87955
MP,Kzz,21,  114.02331
MP,Kxx,22,  113.87955
MP,Kyy,22,  114.45898
MP,Kzz,22,  114.02331
MP,Kxx,23,  113.87955
MP,Kyy,23,  114.45898
MP,Kzz,23,  114.02331
MP,Kxx,24,  114.45898
MP,Kyy,24,  113.87955
MP,Kzz,24,  114.02331
*endif
