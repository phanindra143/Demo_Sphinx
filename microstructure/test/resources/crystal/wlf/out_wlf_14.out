            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\6
             3.00000      -1781557.71676         -7526.33220         -3440.42177
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000           -14.28028           -67.31045
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000         -5791.59079      -1788405.22566         -6978.82688
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000           -21.54422         15625.00000            -6.96418
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000         -4695.75676        -13437.88917      -1779351.29989
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000           -55.21543            42.41714         15625.00000
  /%
